backend/packages/Upgrade/src/api/controllers/ExperimentClientController.ts

```diff
@@ -35,13 +35,7 @@ import { env } from '../../env';
 interface IExperimentAssignment {
   expId: string;
   expPoint: string;
-  twoCharacterId: string;
-  description: string;
-  assignedCondition: {
-    conditionCode: string;
-    twoCharacterId: string;
-    description: string;
-  };
+  assignedCondition: object;
```

Comment:
we should really avoid `object`, it is almost as meaningless as `any` or `unknown`, it will give unexpected warnings that aren't helpful.

If we really have places that we could get an object that could be anything, I see `Record<string, unknown>` as generally recommended best alternative (I have also used `{ [key: string]: unknown }` in places.

But for this, we should be able to give `assignedCondition` an explicit type.